packages/editor/src/components/editor-canvas/index.js

```diff
@@ -348,6 +357,7 @@ function EditorCanvas(
 				<BlockList
 					className={ classnames(
 						className,
+						'is-' + deviceType.toLowerCase() + '-preview',
```

Comment:
@youknowriad I got a ping from @sergiu-radu that we previous had this class on the `.editor-styles-wrapper` container, and since 6.5 on the `.is-root-container` div, so this is a small breaking change. @sergiu-radu, could you maybe give us some more info about why you were relying on this?

Reply:
Hi guys 👋

No, this class was not on `.editor-styles-wrapper` but outside this container...

To be more specific, this class`.is-desktop-preview` was a direct child container of `.edit-post-visual-editor__content-area` container and inside it was the `.editor-styles-wrapper`.
Here is a screenshot - https://share.cleanshot.com/D1h4tF67

Now, in 6.5 the `.is-desktop-preview` class is placed on `.is-root-container` and this breaks a lot of things for us, and I think for other classic themes too.
Here is a screenshot - https://share.cleanshot.com/r57zPg57

We will try to make some tests and adjust our code first but if we won't succeed - will open an issue.

Thank you.

Reply:
To be clear, per the extensibility guidelines of Gutenberg, this is an internal modifier class. It's unfortunate that theme authors are relying on it. We can try to mitigate the issues though (the class will probably end up being present in two places)

Reply:
@youknowriad Why was it needed to move it from outside the canvas to inside the canvas?

Reply:
This PR unified the behavior between post and site editors, which also meant some markup simplification (removing unnecessary wrappers in the post editor)

Reply:
Hey guys, we made some more investigations and yes, the problem seems to not be related to that class actually but something else (most likely because of 2 scenarios - #1 when we have iframe on desktop and #2 when we don't have the iframe)
We are still investigating this issue and will get back with more details.

Thank you!

Reply:
I don't remember exactly but I believe this class was not present at all in the site editor and I just kept it around (for BC) but in a unified place. We can consider moving it to anther place in. `EditorCanvas` if needed but it shouldn't be something that the editor implementor have to think about. The device handling moved within `EditorCanvas`